### PR TITLE
XRT-2128 Update app_component example, coordinated common env var scripts under one file

### PR DESCRIPTION
**Pull request by luke-shields**
Closed old pull request to 2.1
https://github.com/IOTechSystems/xrt-examples/pull/206
And created this new pull request

### CODE REVIEW

**Review by dpazj — APPROVED** [review]
LGTM

File: Bridge/mqtt/pub/README.md [hunk 1]
```diff
@@ -1,29 +1,16 @@
...
 **Run XRT with the config folder:**
 
 This is assuming that the following pre-requisites are satisfied:
 
-* XRT is installed
-* LD_LIBRARY_PATH has been correctly set
-* XRT_LICENSE_FILE has been set to the location of the xrt license
+- XRT is installed
+- LD_LIBRARY_PATH has been correctly set
+- XRT_LICENSE_FILE has been set to the location of the xrt license
```
Comment (BinduRao2018) [inline Bridge/mqtt/pub/README.md:9]: Execution of set_env_vars.sh  missing in pub & sub directory

File: DeviceServices/bacnet-mstp/deployment/state/schedules.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
       "interval": 3000000,
       "name": "schedule1",
       "resource": [
-        ".*"
+        "analog_value_0:present-value",
```
Comment (BinduRao2018) [inline DeviceServices/bacnet-mstp/deployment/state/schedules.json]: Why is the schedule resource changed?

**Review by BinduRao2018 — CHANGES_REQUESTED** [review]
Instruction to execute set_env_vars.sh missing in some examples readme.txt.

**Review by BinduRao2018 — APPROVED** [review]
LGTM

**Pull request merged**